app/team.go

```diff
@@ -1381,8 +1381,9 @@ func (a *App) PermanentDeleteTeam(team *model.Team) *model.AppError {
 	}
 
 	if channels, err := a.Srv().Store.Channel().GetTeamChannels(team.Id); err != nil {
-		if err.Id != "app.channel.get_channels.not_found.app_error" {
-			return err
+		var nfErr *store.ErrNotFound
+		if !errors.As(err, &nfErr) {
+			return model.NewAppError("PermanentDeleteTeam", "app.channel.get_channels.get.app_error", nil, err.Error(), http.StatusInternalServerError)
```

Comment:
`nfErr.Error()`

Reply:
This case is ok, notice the `!` before `errors.As(...`

Reply:
Ah my bad.